components/base/kinematicbase/fake_kinematics.go

```diff
@@ -63,11 +66,19 @@ func (fk *fakeKinematics) Kinematics() referenceframe.Frame {
 }
 
 func (fk *fakeKinematics) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
+	fk.lock.Lock()
+	defer fk.lock.Unlock()
 	return fk.inputs, nil
 }
 
 func (fk *fakeKinematics) GoToInputs(ctx context.Context, inputs []referenceframe.Input) error {
 	_, err := fk.planningFrame.Transform(inputs)
-	fk.inputs = inputs
+	fk.lock.Lock()
+	fk.inputs = []referenceframe.Input{
+		{Value: fk.inputs[0].Value + inputs[0].Value},
+		{Value: fk.inputs[1].Value + inputs[1].Value},
+	}
+	defer fk.lock.Unlock()
+	time.Sleep(150 * time.Millisecond)
```

Comment:
Why is this here?
This smells like a patch for a race condition that should be fixed elsewhere.

Reply:
Its not a fix for a race condition its me trying to simulate the base taking some amount of time to get the location it needs to get to so replan events can trigger

Reply:
Makes sense. Can we add a comment saying that's what it's for?